Documentation/devel/stage1-implementors-guide.md

```diff
@@ -104,12 +104,25 @@ For example, it removes the network namespace of a pod.
 * `--debug` to activate debugging
 * UUID of the pod
 
+### `rkt stop` => "coreos.com/rkt/stage1/stop"
+
+The optional stop entrypoint initiates an orderly shutdown of stage1.
+
+In the bundled rkt stage 1, the entrypoint is calling `systemctl halt` on the container (through SSH).
+
+This entrypoint was added in interface version 3.
```

Comment:
Since this entrypoint is optional, we don't need to bump the interface version for it. There was indeed no need to test the interface version 3 in the code: using getStage1Entrypoint is enough, as you did:
```
ep, err := getStage1Entrypoint(dir, stopEntrypoint)
```

@krnowak do you agree?